src/lib/utils.js

```diff
@@ -13,6 +15,42 @@ function removeFalsyEntries (object = {}) {
     }, {})
 }
 
+/**
+ *
+ * @param {number} number
+ * @returns string representation of the number in spanish words
+ */
+function convertNumberToTextInSpanish (number) {
+  const ConverterClass = conversor.conversorNumerosALetras
+  const myConverter = new ConverterClass()
+  const numberWord = myConverter.convertToText(number)
+  const capitalLetter = numberWord.slice(0, 1).toUpperCase()
+  const restOfLetters = numberWord.slice(1)
+
+  return `${capitalLetter}${restOfLetters}`
+}
+
+/**
+ *@param {number} number
+ * format the number in the correct currency
+ *
+*/
+function formatCurrency (value) {
+  var numberToString = value.toString()
+  if (numberToString.length === 4) {
+    const thousand = numberToString.slice(0, 1)
+    const restOfValues = numberToString.slice(1)
+    return `$${thousand},${restOfValues}.00`
+  }
+  const formatter = new Intl.NumberFormat('es-MX', {
+    style: 'currency',
+    currency: 'MXN'
+  })
+  return formatter.format(value)
+}
```

Comment:
Asi te evitas la logica extra
```suggestion
function formatCurrency (value) {
  const formatter = new Intl.NumberFormat('es-MX', {
    style: 'currency',
    currency: 'MXN',
    maximumSignificantDigits: 3
  })
  return formatter.format(value)
}
```